flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": [ "arm", "aarch64", "x86_64" ]
```

Comment:
you can remove this file. only supported arch are "aarch64", "x86_64" so it is redundant.